### PR TITLE
Update rubygem-ldap_fluff to 0.5.0 - RPM

### PR DESCRIPTION
**Pull request by lzap**
For plugin updates, please indicate which repos this should be built into:

* Nightly

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
Needed for https://github.com/theforeman/foreman/pull/8205

**Review by tbrisker — COMMENTED** [review]
not sure why tests didn't complain, but the gem now requires net-ldap >= 0.11 while the rpmspec only has 0.3.1

**Comment by ezr-ondrej** [comment]
> not sure why tests didn't complain, but the gem now requires net-ldap >= 0.11 while the rpmspec only has 0.3.1

Yeah, would be worth to bump it.

**Review by tbrisker — DISMISSED** [review]
@lzap can you please update the requires to match the gem?

**Comment by tbrisker** [comment]
I fixed the versions and updated the spec to use the template so it auto-updates next time. repoclosure is unrelated (fog-ovirt bump)

File: packages/foreman/rubygem-ldap_fluff/rubygem-ldap_fluff.spec [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
-Requires: %{?scl_prefix}rubygem(net-ldap) >= 0.3.1
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby >= 2.4.0
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
-BuildRequires: %{?scl_prefix_ruby}rubygem(rake)
 BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
+
 %{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
```
Comment (evgeni) [inline packages/foreman/rubygem-ldap_fluff/rubygem-ldap_fluff.spec:30]: mind droping this line?

**Review by evgeni — APPROVED** [review]
ACK (not merging to give Tomer a chance to address the stray comment)

File: packages/foreman/rubygem-ldap_fluff/rubygem-ldap_fluff.spec (hunk 1 above)
Comment (tbrisker) [reply packages/foreman/rubygem-ldap_fluff/rubygem-ldap_fluff.spec]: done

**Review by evgeni — APPROVED** [review]

**Pull request merged**